modules/phase_field/include/ics/MultiSmoothSuperellipsoidIC.h

```diff
@@ -31,29 +31,50 @@ class MultiSmoothSuperellipsoidIC : public SmoothSuperellipsoidBaseIC
 
   virtual void initialSetup();
 
-
 protected:
   virtual void computeSuperellipsoidSemiaxes();
   virtual void computeSuperellipsoidCenters();
   virtual void computeSuperellipsoidExponents();
+  virtual Real overlapCheck(const Point & newcenter, const Real & _nc_as, const Real & _nc_bs, const Real & _nc_cs, const Real & _nc_ns);
 
+  unsigned int _numtries;
   unsigned int _numbub;
   Real _bubspac;
-
-  unsigned int _numtries;
   Real _exponent;
   Real _semiaxis_a;
   Real _semiaxis_b;
   Real _semiaxis_c;
   Real _semiaxis_a_variation;
   Real _semiaxis_b_variation;
   Real _semiaxis_c_variation;
+  Real _nv;
```

Comment:
These parameters are just used in the code to store the size of each vector input parameter (_numbub, semiaxis_a, etc) before checking that each vector is the same size, rather then checking and rechecking the vector size inside the for loop arguments. I did not initialize them in the constructor because i thought i only needed user defined parameters to be there.